packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonList.tsx

```diff
@@ -0,0 +1,64 @@
+import { css, cx } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+
+import { useStyles2 } from '../../../themes';
+
+import { RadioButtonDot } from './RadioButtonDot';
+
+export interface RadioButtonListProps<T> {
+  name: string;
+  id?: string;
+  options: Array<SelectableValue<T>>;
+  keySelector: (option: T) => string;
```

Comment:
Suggestion: I think we could use a random id generator here, is there any specific reason why we would want the ID to be controlled by the user? It serves no other purpose besides making sure each ID is unique in a radio group 🤔